build.gradle

```diff
@@ -24,19 +24,63 @@ plugins {
 
 allprojects {
     repositories {
```

Comment:
The following files also have `repositories` declarations: in theory, we should cover them too:
- build.gradle (this file, the `buildscript` block)
- settings.gradle
- buildSrc/build.gradle.kts